static/chats.js

```diff
@@ -56,20 +65,25 @@ class Zinja {
     this.vards = vards;
     this.zinja = zinja;
     this.laiks = laiks;
-  }
-
-  formateRindu() {
+    }
+    paraditNoskanojumu() {
+      if (Object.entries(garaStavoklisNoServera) == 0) return ''
```

Comment:
Te varētu atgriezt tomēr: "-" vai "*", nezinu kas labāk.
Respektīvi:
```js
if (Object.entries(garaStavoklisNoServera) == 0) return '-'

Reply:
Jā, kodā ieliku, ka tiek atgriezts "-", ja garaStavoklisNoServera ir tukšs, tukšs šis mainīgais ir tikai tajā momentā, kamēr nav atnākuši dati "mood" no servera